content/en/terraform-samples.md

```diff
@@ -0,0 +1,243 @@
+---
+title: "Effective Terraform Samples"
+linkTitle: "Terraform Style Guide"
+type: docs
+weight: 20
+---
+
+## Introduction {#introduction}
```

Comment:
Before offering a detailed review, I'm first considering whether having a separate page so similar to the primary style guide is a good idea.

* The text will fall out of sync
* The scope and context around guidelines will not benefit from easily "triangulating" with other languages

It seems on a first review like it would be better to use Terraform tabs on the main guide, replacing the definite use of code highlights with commentary & code as needed.

I know I had previous feedback that a split may be helpful, but on understanding the proposed content of the style guide better I don't see as significant a gap between Terraform guidance and the other languages as I might have expected.